### ISSUE TITLE
Make it possible to switch all tabs on a page

### ISSUE THREAD

**Issue by drewhk** [lightbend/paradox#11]
As a Java developer, I would find it very annoying to flip all tabs one-by-one to Java for example.

**Comment by eed3si9n** [comment]
+1

**Comment by pvlugter** [comment]
Yes, we've wanted to add this too. Even better if it remembered a preference for the whole documentation site.

In the cinnamon documentation, we've also used tabs in a different way (from build tool or programming language) where we have the minimal configuration in the main tab and the possible defaults from the reference.conf in a secondary tab. We've found this to be useful. And in this case, we don't want to switch all tabs on a page or remember a preference. Examples on this page:

http://monitoring.lightbend.com/docs/2.0.x/plugins/chmetrics/reporters.html

So if this feature could have switching all tabs by default, but possible to turn off for something like the configuration tabs.
